sentry/src/main/java/io/sentry/ISentryClient.java

```diff
@@ -47,6 +47,8 @@ public interface ISentryClient {
    */
   void flush(long timeoutMillis);
 
+  void flushLogs(long timeoutMillis);
```

Comment:
Not the biggest fan of adding a new API here, an alternative would be to move this directly into `ILoggerApi`